docs/source/index.rst

```diff
@@ -1,9 +1,5 @@
-PyDAX Documentation
-===================
-
-PyDAX is a simple Python API for downloading and loading datasets from IBM's `Data Asset Exchange`_.
-
-.. _Data Asset Exchange: https://developer.ibm.com/exchanges/data/
+.. include:: ../../README.rst
```

Comment:
The documentation index page is a bit crowded when adding the README content, but I think it will be ok in the future if we condense the quick start guide into a separate user guide. What do you think?

Reply:
Yeah, I agree. It is important to have the concise information on the front page to attract impatient users. We can split README to multiple sections in the future and selectively insert them at different places on the front page.

Reply:
gotcha, ok good idea